service.go

```diff
@@ -379,7 +379,10 @@ func (client *Client) ListServices(variables *PayloadVariables) (*ServiceConnect
 	return &q.Account.Services, nil
 }
 
-func (client *Client) ListServicesWithFilter(filterIdentifier string, variables *PayloadVariables) (*ServiceConnection, error) {
+func (client *Client) ListServicesWithFilterId(filterId string, variables *PayloadVariables) (*ServiceConnection, error) {
```

Comment:
lets not change the interface of the client.  We can kep the same function and just return the error if they don't present an ID with a message that the API currently doesn't support this.